locals.tf

```diff
@@ -35,12 +35,32 @@ locals {
 
   cool_shared_services_cidr_block = data.terraform_remote_state.sharedservices_networking.outputs.vpc.cidr_block
 
-  guacamole_fqdn = format("guac.%s.%s", var.assessment_account_name, var.cool_domain)
+  guacamole_fqdn = format("guac.%s.%s", local.assessment_account_name_base, var.cool_domain)
```

Comment:
This works for Production environments, but for non-Production environments, I think we want it to be: 
`format("guac.%s.%s.%s", local.assessment_account_name_base, local. workspace_type, var.cool_domain)`

Reply:
In the case of staging, for example, `var.cool_domain` is `staging.cool.cyber.dhs.gov`.

Reply:
Yup, I realized this 2 seconds after I posted it and was just about to confirm that.  Thanks for clarifying.